sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -267,8 +235,8 @@ def get_directory_client(self, directory_path=None):
         :rtype: ~azure.storage.fileshare.ShareDirectoryClient
         """
         _pipeline = Pipeline(
-            transport=TransportWrapper(self._pipeline._transport), # pylint: disable = protected-access
-            policies=self._pipeline._impl_policies # pylint: disable = protected-access
+            transport=TransportWrapper(self._pipeline._transport),  # pylint: disable = protected-access
+            policies=self._pipeline._impl_policies  # pylint: disable = protected-access
```

Comment:
For future reference but I think the convention we've generally seen for disable is no spaces after the disable statement (except for multiple disable lines), so this would look like:
`  # pylint: disable=protected-access`

Would be nice to get them to all match, but I know I am guilty of not following this convention sometimes as well, but we can make this a best-effort sort of thing

Reply:
Sorry I missed this go-around, will fix it :)